packages/amplify-util-mock/src/__tests__/velocity/model-auth.test.ts

```diff
@@ -1574,9 +1574,9 @@ describe('with identity claim feature flag disabled', () => {
       expect(listAuthVTLRequest.stash.authFilter).toEqual(
         expect.objectContaining({
           or: expect.arrayContaining([
-            expect.objectContaining({ child: { eq: ownerRequest.jwt['cognito:username'] } }),
-            expect.objectContaining({ parent: { eq: ownerRequest.jwt['cognito:username'] } }),
-          ]),
+            expect.objectContaining({ child: { in: expect.arrayContaining([ownerRequest.jwt["cognito:username"]]) } }),
+            expect.objectContaining({ parent: { in: expect.arrayContaining([ownerRequest.jwt["cognito:username"]]) } })
+          ])
```

Comment:
this change is likely going to fail build.
make sure that `yarn build-and-verify` passes locally.